dist/45e582c10309a2b2f5f0.js

```diff
@@ -27,18 +25,10 @@ mealArea.addEventListener('click', async (e) => {
     showInfo(metricsInfo, uniqueID);
   }
 });
+
 const popupWindow = document.querySelector('.popup-window');
 popupWindow.addEventListener('click', (e) => {
   if (e.target && e.target.matches('i.minimize-card')) {
     popupWindow.innerHTML = '';
   }
 });
-
-document.addEventListener('click', (e) => {
-  const { target } = e;
-  if (target.matches('.comment')) {
-    const { strMealThumb, strMeal } = foodDescr[comment.indexOf(target)];
-    popWindow.innerHTML = showPopup(strMealThumb, strMeal);
-    togglePopup();
-  } else if (target.matches('#close')) togglePopup();
-});
```

Comment:
You could have added another element listener in the event delegation, or you could have just created a new event rather than deleting my code.

Reply:
You have done a great job :+1: But you have to make a few changes. Check the above comments